### PR TITLE
Add an example for galaxyline

### PR DESCRIPTION
**Pull request by zawedcvg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,32 @@ require("lualine").setup({
...
+	
+### [galaxyline](https://github.com/glepnir/galaxyline.nvim)
+
+<details>
+<summary>An example galaxyline setup </summary>
+
+```lua
+local navic = require("nvim-navic")
+local gl = require("galaxyline")
+local condition = require("galaxyline.condition")
```
Comment (SmiteshP) [inline README.md:195]: I don't think this is required. Also remove the extra empty line.

**Review by SmiteshP — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zawedcvg) [reply README.md:195]: line 202 will not work without this variable

Comment (SmiteshP) [reply README.md:195]: Nope, there `condition` is a key in the table `nvimNavic`. It does not rely on the import statement. The imported condition is for using built-in conditions like condition.buffer_not_empty etc.

Comment (zawedcvg) [reply README.md]: I see. Removed the variable now.

**Review by SmiteshP — APPROVED** [review]
Thanks 👍🏽

**Pull request merged**